test/full_url_for.spec.js

```diff
@@ -2,7 +2,12 @@
 
 describe('full_url_for', () => {
   const ctx = {
-    config: {}
+    config: {
+      url: 'http://example.com',
+      external_link: {
+        exclude: []
+      }
```

Comment:
Need condition checking in `isExternalLink()` to skip `exclude` checking, if `config.external_link` doesn't exist.

Reply:
Fixed in https://github.com/hexojs/hexo-util/pull/124/commits/34088245fc4f576321546a7dab4cea10eb8ffd74 (#124)